### PR TITLE
fix v \s\s 1 2 3

### PR DESCRIPTION
**Pull request by 514366607**
Error By 3ds Max Wavefront OBJ Exporter

### CODE REVIEW

File: decoder.go [hunk 1]
```diff
@@ -30,6 +31,8 @@ func Decode(r io.Reader) (*OBJFile, error) {
 
 	scanner := bufio.NewScanner(r)
 	for scanner.Scan() {
+		re, _ := regexp.Compile(`\s+`)
```
Comment (arl) [inline decoder.go]: Would have preferred another way to check that an regex, but why not.
In any case, the regex compilation should be done outside of the loop, and preferably once for all, with a `regex.MustCompile` global block

**Review by arl — CHANGES_REQUESTED** [review]
Thank you for that.
Can we move the regex compilation our of the Decode path? Then I'll merge it.

**Comment by 514366607** [comment]
Submitted, please review.

**Review by arl — APPROVED** [review]
LGTM

Thank you🎉

**Pull request merged**